crates/tx-parser/src/filter_types.rs

```diff
@@ -54,3 +55,19 @@ impl TxFilterConfig {
         })
     }
 }
+
+/// Wrapper to hold Inscription Data after parsing and filtering.
+#[allow(clippy::large_enum_variant)]
+pub(crate) enum FilteredInscriptionType {
+    Checkpoint(SignedBatchCheckpoint),
+    DA(Vec<u8>),
+}
```

Comment:
What does this do?

Reply:
This was a shim enum, which facilitated conversion to ProtocolOp. I have removed it